### PR TITLE
Added filter to Admin>Core>Delta filtering delta jobs on status pendi…

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
…ng+started or not.

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2558 Add a filter by finilized/non finilized deltas in Admin UI](https://app.clickup.com/t/2192114/QF-2558)

**Review by suricactus — APPROVED** [review]
Amazing!

File: docker-app/qfieldcloud/core/admin.py [hunk 1]
```diff
@@ -727,6 +728,24 @@ def has_delete_permission(self, request, obj):
...
+            ("finalized", _("finalized")),
+            ("not finalized", _("not finalized")),
+        )
+
+    def queryset(self, request, queryset):
+        q = Q(last_status="pending") | Q(last_status="started")
+        if self.value() == "not finalized":
+            return queryset.filter(q)
+        else:
+            return queryset.filter(~q)
```
Comment (suricactus) [inline docker-app/qfieldcloud/core/admin.py:746]: @why-not-try-calmer hey, actually this is never checking if the selected filter is ALL and always filter for either `finilized` or `not finalized`.

Comment (why-not-try-calmer) [reply docker-app/qfieldcloud/core/admin.py:746]: What is `ALL`? `ALL` == "the user is not applying the filter at all" ? Or is it the disjunction of all statuses?

**Pull request merged**